src/OrchardCore.Modules/OrchardCore.Media/Models/MediaProfilesDocument.cs

```diff
@@ -1,19 +1,21 @@
 using OrchardCore.Data.Documents;
+using Format = OrchardCore.Media.Processing.Format;
+using ResizeMode = OrchardCore.Media.Processing.ResizeMode;
 
 namespace OrchardCore.Media.Models;
 
 public class MediaProfilesDocument : Document
 {
-    public Dictionary<string, MediaProfile> MediaProfiles { get; init; } = new Dictionary<string, MediaProfile>(StringComparer.OrdinalIgnoreCase);
+    public Dictionary<string, MediaProfile> MediaProfiles { get; init; } = new(StringComparer.OrdinalIgnoreCase);
 }
 
 public class MediaProfile
 {
     public string Hint { get; set; }
     public int Width { get; set; }
     public int Height { get; set; }
-    public Processing.ResizeMode Mode { get; set; }
-    public Processing.Format Format { get; set; }
+    public ResizeMode Mode { get; set; }
+    public Format Format { get; set; }
```

Comment:
I called Rider's "Organize imports" feature to get rid of dead `using` namespaces and it did this. I think it looks better, but comment here if I should revert them.